contracts/src/core/EigenDAServiceManagerStorage.sol

```diff
@@ -37,6 +37,13 @@ abstract contract EigenDAServiceManagerStorage is IEigenDAServiceManager {
      * this is the percentage of the total stake needed to confirm a blob
      */
     bytes public constant quorumConfirmationThresholdPercentages = hex"4242";
+
+    /** 
+     * @notice The quorum numbers required for confirmation stored as an ordered bytes array 
+     * these quorum numbers have respective canonical thresholds in the
+     * quorumConfirmationThresholdPercentages and quorumAdversaryThresholdPercentages above
+     */
+    bytes public constant quorumNumbersRequired = hex"0001";
```

Comment:
Is this requiring quorums 0 and 1?
Should we start with just quorum 0 and update it when we introduce second required quorum?

Reply:
I'm not sure how much this matters. @ian-shim if you feel strongly about this, please just go ahead and give some direction. I'm fine with it either way.